packages/onchainkit/src/utils/formatter.ts

```diff
@@ -1,4 +1,14 @@
-export function formatAmount(num: string): string {
+import { getRoundedAmount } from '@/internal/utils/getRoundedAmount';
+
+export const formatUSD = (amount: string) => {
+  if (!amount || amount === '0') {
+    return null;
+  }
+  const roundedAmount = Number(getRoundedAmount(amount, 2));
+  return `~$${roundedAmount.toFixed(2)}`;
+};
+
+export const formatMaybeScientificNotationToDecimal = (num: string) => {
```

Comment:
nit: This isn't that important, but because this is exported to the public, this function name is making me 🤨

I think `formatAmount` was probably fine, especially if it had a TSDoc comment. If you really wanna be explicit, something like "toDecimalString" would be good too.